### DISCUSSION TITLE
Combining webpack chunks into larger, fewer bundles?

### DISCUSSION THREAD

**Discussion by tomjakubowski** [jupyterlite/jupyterlite#1382] [Q&A]
Hello,

I am trying to reduce the file count in a jupyterlite site. After running `jupyter lite build`, the contents of `build/` in the output look like they come from a Webpack build: each file N.<sha>.js adds a chunk of code to the output like this:

```
"use strict";(self.webpackChunk_JUPYTERLAB_CORE_OUTPUT=self.webpackChunk_JUPYTERLAB_CORE_OUTPUT||[]).push([[4498],{4498: function(…
```

Is it possible, or advisable, to override the webpack config used to produce this output, in order to produce a bundle with fewer output files?

**Comment by jtpio** [comment]
The `jupyterlite-core` package contains a pre-built archive with all the static assets. That archive is built from https://github.com/jupyterlite/jupyterlite/tree/main/app at release time.

The `jupyter lite build` command contains a `--app-archive` option though. So maybe it could be worth to try building https://github.com/jupyterlite/jupyterlite/tree/main/app  from source after modifying the webpack config, and then use that archive for the `jupyter lite build` command.
